src/ucp/wireup/wireup.h

```diff
@@ -44,7 +44,12 @@ typedef struct ucp_wireup_msg {
     ucp_rsc_index_t               dst_aux_index;    /* Index of receiver wireup resource */
     uint16_t                      flags;            /* Wireup flags */
     uint8_t                       addr_len;         /* Length of first address */
-    /* addresses follow */
+    uint8_t                       peer_name_len;    /* Length of peer name */
```

Comment:
I would protect it for a two reasons:
1) to make sure that people don't use it in production code
2) clearly marks the intend of the variable

Obviously this is not critical issue, but addressing the above will make
code cleaner